### DISCUSSION TITLE
Multilanguage blog

### DISCUSSION THREAD

**Discussion by dmasiar** [facebook/docusaurus#3345] [General]
Hey, I would like to run Docusaurus in blog-only mode (docs are on different domain) and I would like to leverage the i18n functionality. Is there any way I can make blog in multiple languages?

Thanks!
Dan

**Comment by slorber** [comment]
Hi,
v1 or v2?

We don't invest too much on v1 anymore. 
v2 will be ready once we release i18n (work in progress, RFC: https://github.com/facebook/docusaurus/issues/3317)

So far this is not something on the short term roadmap, but we could add i18n support to the blog too (v2).

**Reply by dmasiar** [reply]
Hi, thanks for your answer. 

I am currently using V1 but if this would be possible in V2, there is no problem to switch. The question was somewhat more generic - whether it is possible to have multilingual blog in Docusaurus at all (so for example when user switches from English to French, the blog would take articles from blog/fr instead of blog/en or something similar).

Thanks!

**Reply by slorber** [reply]
Hi, yes I'm currently working on v2 i18n and being able to translate the blog is indeed a goal.
